core/domain/skill_domain.py

```diff
@@ -126,7 +148,8 @@ def __init__(
             notes: str. General advice for creators about the
```

Comment:
Is this html as well? If so, say so and pass it through html_cleaner.clean().

Reply:
Yeah, done.